VM-RIght-Sizing/manifest.yaml

```
@@ -0,0 +1,29 @@
+manifestVersion: 1.0
+projects:
+  - name: jira-connection
+    path: first-example
+  - name: slack-connection
+    path: second-example
+  - name: workflow-calculate-ticket
+    path: third-example
+
+environmentGroups:
+  - name: production
+    environments:
+      - name: production-environment
+        url:
+          value: "" # Enter environment ID e.g. https://<tenantId>.apps.dynatrace.com/
```

URLs can be defined as environment variables as well. 
As you're asking users to fill out the shell scrip already I think it'd make it easier if they can just define their URL there as well :) 

```suggestion
          type: environment
          value: DT_URL
```

and then including the DT_URL variable and description how it should look in `input-needed.sh`